libeventheader-tracepoint/samples/CMakeLists.txt

```diff
@@ -1,15 +1,24 @@
-add_executable(dynamic-sample
+add_executable(eventheader-dynamic-sample
     dynamic-sample.cpp)
-target_link_libraries(dynamic-sample
+target_link_libraries(eventheader-dynamic-sample
     PUBLIC eventheader-tracepoint tracepoint)
-target_compile_features(dynamic-sample
+target_compile_features(eventheader-dynamic-sample
     PRIVATE cxx_std_17)
 
 add_executable(eventheader-sample
     sample.cpp)
 target_link_libraries(eventheader-sample
     PUBLIC eventheader-tracepoint tracepoint)
 
+add_executable(eventheader-tracepoint-sample
```

Comment:
Note: Add a new sample. Existing "eventheader-sample" shows basic usage of TraceLoggingWrite. New "eventheader-tracepoint-sample" exercises every eventheader type, so that you can collect a perf.data trace and test decoding it.